src/Microsoft.SymbolStore/SymbolStoreKey.cs

```diff
@@ -32,18 +32,30 @@ public sealed class SymbolStoreKey
         /// </summary>
         public static SymbolStoreKey[] EmptyArray = new SymbolStoreKey[0];
 
+        /// <summary>
```

Comment:
Don't you need this to be a collection of `VsPdbChecksum` instead -- a PDB can have multiple checksums.